.github/workflows/enos-ci-bootstrap-ent.yml

```diff
@@ -0,0 +1,44 @@
+name: enos-ci-bootstrap-ent
+
+# These permissions are necessary for the doormat-action to get the OIDC token
+permissions:
+  contents: read
+  id-token: write
+
+on:
+  workflow_call:
+
+jobs:
+  bootstrap-ci-ent:
+    env:
+      TF_WORKSPACE: "${{ github.event.repository.name }}-ci-enos-bootstrap"
+      TF_VAR_repository: ${{ github.event.repository.name }}
+      TF_VAR_aws_ssh_public_key: ${{ secrets.SSH_KEY_PUBLIC_CI }}
+      TF_TOKEN_app_terraform_io: ${{ secrets.TF_API_TOKEN }}
+    runs-on: ubuntu-latest
```

Comment:
I know we'll be moving this file to another PR, but we'll need to change the runner to use self-hosted runners for private/internal accounts.